### PR TITLE
Edit the routes

### PR DESCRIPTION
**Pull request by isaka-lumato**
### ARTICLES/BLOG

- Edited the routes into resourceful routing

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root "articles#index"
-  get "articles/index"
-  get "/articles/:id", to: "articles#show"
+  resources :articles
```
Comment (vic778) [inline config/routes.rb:3]: kindly comment or delete these lines and change to `resources :articles` this is the best way to make root in ruby. in rails the best way link to a title or something is to use ~<%= link_to +path + the action%> so it gonna be `<h1>Articles</h1>
<ul>
<%@articles.each do |article|%>

        <%= link_to article.title, article %>
     
<li><%= article.body%></li>
<%end%>
</ul>~

**Review by vic778 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @isaka-lumato , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Descriptive commits and correct use of the Git Flow :fire:
- [x] :white_check_mark: page titles :white_check_mark: text headings :white_check_mark: basic structure of the page 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: config/routes.rb (hunk 1 above)
Comment (isaka-lumato) [reply config/routes.rb:3]: thanks for the feedback,. am gonna work on the requested changes ASAP

File: app/views/articles/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,7 @@
 <h1>Articles</h1>
 <ul>
 <%@articles.each do |article|%>
-<li>  <a href="/articles/<%= article.id %>">
-        <%= article.title %>
+<li>       <%= link_to article.title, article %>
       </a></li>
```
Comment (vic778) [inline app/views/articles/index.html.erb:5]: kindly remember to delete the `</a>

**Review by vic778 — APPROVED** [review]
## hey @isaka-lumato 
Gret job, nothing ekse to say. it;s time to merge your branch and go-ahead

**Pull request merged**